packages/dataviews/src/dataviews-layouts/utils/get-clickable-item-props.ts

```diff
@@ -31,5 +35,7 @@ export default function getClickableItemProps< Item >( {
 				onClickItem( item );
 			}
 		},
+		...( id ? { id } : {} ),
+		...( ariaLabelledby ? { 'aria-labelledby': ariaLabelledby } : {} ),
```

Comment:
Why are we adding props and just returning them instead of just adding these props directly to the elements?

Reply:
Hi @youknowriad, because of a bad night of sleep, I guess 😅 I added the props directly to the elements.